libs/sdk-core/src/breez_services.rs

```diff
@@ -929,6 +933,47 @@ impl BreezServices {
             }
         });
     }
+
+    fn init_logging(working_dir: &str) -> SdkResult<()> {
```

Comment:
Will this work side by side with the binding logger that we initialize in uniffi_binding.rs? Or one will override another?

Reply:
By looking at the failed CI tests

```
thread '<unnamed>' panicked at 'Builder::init should not be called after logger initialized: SetLoggerError(())', /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/env_logger-0.9.3/src/lib.rs:816:14
```

it looks like my change conflicts with the `uniff_binding` log stream.

Looking into ways to make them compatible.

Reply:
The main way to support multiple global loggers is https://crates.io/crates/multi_log . However that crate is not maintained since 5 years.

Instead I'm looking into using `try_init` which should fail gracefully if another global logger, like the `uniffi` logstream, got initialized earlier.